projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/_all-sites-menu.scss

```
@@ -0,0 +1,38 @@
+.admin-color-blue,
+.admin-color-coffee,
+.admin-color-ectoplasm,
+.admin-color-fresh,
+.admin-color-light,
+.admin-color-midnight,
+.admin-color-modern,
+.admin-color-ocean,
+.admin-color-sunrise {
+	/**
+	 * Styles for All Sites menu
+	 */
+	#wpadminbar .wp-admin-bar-all-sites a.ab-item {
+		display: flex;
+		align-items: center;
+	}
+	#wpadminbar .wp-admin-bar-all-sites a.ab-item svg {
+		fill: $menu-icon;
+		margin-right: 4px;
+	}
+	#wpadminbar .wp-admin-bar-all-sites:hover a.ab-item svg {
+		fill: $menu-submenu-focus-text;
+	}
```

> For Core's color schemes (Default(Fresh), Light, Modern, Blue, Coffee, Ectoplasm, Midnight, Ocean, Sunrise)
> I introduced new CSS files for each color scheme.

I think we can avoid this whole thing by just using `fill: currentColor;` trick. It will pick up the current color that the existing CSS already set up for dashicons:

```suggestion
	#wpadminbar .wp-admin-bar-all-sites a.ab-item svg {
		fill: currentColor;
		margin-right: 4px;
	}
```

If we add this CSS as inline style, we can remove ALL `masterbar/admin-color-schemes/colors/*/all-sites-menu.scss` that are added in this PR. What do you think?

---

The color of the icon and the color of the text are different in some color schemes, so we cannot directly apply `currentColor` to the SVG tag in HTML. 

However, we should be able to leverage `currentColor` by the following;

1. Apply `content: ""; width: XXpx; height: YYpx;` to the `.ab-item:before` pseudo-element to utilize [the Core's color styles](https://github.com/WordPress/wordpress-develop/blob/8d0aed455b1790c5d51386f7675d9e8d68e48edf/src/wp-admin/css/colors/_admin.scss#L400-L403).
1. Apply `mask-image: url("data:image/svg+xml;base64,PHN2ZyB2aWV3Q...")` (by using data url representation of SVG) combined with `background-color: currentColor;`. This configuration applies the icon’s shape as a mask and utilizes the current icon's color from the Code side.

While somewhat tricky I think, this method eliminates the need to maintain icon colors from Core, allowing us to remove `colors/*/all-sites-menu.scss`. I'll go for this approach. Sounds okay for you, @fushar?

---

https://github.com/Automattic/jetpack/pull/36632/commits/03d413542d0e5842f8bd677973d717b7220dd172

---

I think that's clever :)